packages/server/README.md

```diff
@@ -186,17 +186,28 @@ A name is a logical identifier which can be used to associate clients with a par
 If a name has no associated provider, the global provider is used.
 
 ```ts
-import { OpenFeature } from "@openfeature/js-sdk";
+import { OpenFeature, InMemoryProvider } from "@openfeature/js-sdk";
```

Comment:
I've made this look pretty much identical to the Java readme.

Reply:
We should either release this right after it's merged or wait to update the docs until after the feature is live.

Reply:
Ya I plan on releasing immediately after this is merged